packages/psss/src/containers/Modals/ArchiveAlertModal.js

```diff
@@ -44,7 +44,7 @@ const STATUS = {
 export const ArchiveAlertModal = ({ isOpen, onClose, alertId }) => {
   const [status, setStatus] = useState(STATUS.INITIAL);
   const { setIsOpen } = useContext(AlertsPanelContext);
-  const [archiveAlert, { error }] = useArchiveAlert(alertId);
+  const { mutate: archiveAlert, error } = useArchiveAlert(alertId);
```

Comment:
`useMutation` now returns an object rather than an array, see: https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-3#usemutation-now-returns-an-object-instead-of-an-array